src/plugins/kibana_utils/server/index.ts

```diff
@@ -16,5 +16,9 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-
+import { KibanaUtilsPlugin } from './plugin';
 export { Get, Set, createGetterSetter, url } from '../common';
+
+export function plugin() {
+  return new KibanaUtilsPlugin();
+}
```

Comment:
please use `scripts/generate_plugin `to generate valid file structure., 

I prefer to see something like: 

```
import { PluginInitializerContext } from 'src/core/server';
import { KibanaUtilsPlugin } from './plugin';

//  This exports static code and TypeScript types,
//  as well as, Kibana Platform `plugin()` initializer.

export function plugin(initializerContext: PluginInitializerContext) {
  return new KibanaUtilsPlugin(initializerContext);
}

export { KibanaUtilsPluginSetup, KibanaUtilsPluginStart } from './types';

```

Reply:
done